src/types/schema.ts

```diff
@@ -88,3 +88,13 @@ export interface TranslationRequest {
   languages: string[];
   interest_ids: UUID[];
 }
+
+export interface LanguageSupport {
+  id: UUID;
+  title?: string;
+  summary?: string;
+  is_remote?: boolean;
+  deadline: string;
+  language: string;
+  num_pages?: number;
```

Comment:
can we also add the `listing_type` field? It should be a string type, and this should be set with a question on the retool form asking what type of language support request they are uploading

Reply:
done!